Network/openvpn.5m.sh

```diff
@@ -0,0 +1,274 @@
+#!/usr/bin/env bash
+# <bitbar.title>MacOS OpenVPN client & DNS Configuration</bitbar.title>
+# <bitbar.version>v0.88008</bitbar.version>
+# <bitbar.author>glowinthedark</bitbar.author>
+# <bitbar.author.github>glowinthedark</bitbar.author.github>
+# <bitbar.desc>OpenVPN GUI & DNS Manager</bitbar.desc>
+# <bitbar.image>https://telegra.ph/file/7e93fd31b281c9cdcab73.png</bitbar.image>
+# <bitbar.dependencies>openvpn</bitbar.dependencies>
+# <bitbar.abouturl>https://github.com/glowinthedark/bitbar-plugins/blob/master/Network/openvpn.5m.sh</bitbar.abouturl>
+
+# *** CONFIGURATION ***
+
+# - INSTALL OPENVPN:
+
+#     brew install openvpn
+# OPTIONAL:
+#     brew install oath-toolkit (for 2FA token generation)
+
+# NOTE: To avoid having to enter the root password allow
+# passwordless sudo for your user on openvpn, killall and pkill:
+
+# 1. Type in terminal: sudo visudo
+# 2. Append the following line at the end of the file:
+
+# replace_with_your_username        ALL = NOPASSWD: /usr/local/sbin/openvpn, /usr/bin/killall, /usr/bin/pkill
+
+# 3. Search and replace all __TODO__ lines with your specific commands
+# 4. [ON FIRST RUN ONLY] Run the script and grant MacOS accessiblity permissions to the terminal.app, bitbar (or xbar)
+#    and to `and to /usr/bin/osascript`
+
+# MacOS Accessibility Permissions
+# ===============================
+
+# To enable MacOS system accessibilty permissions run (once) from the terminal:
+# osascript -e 'tell application "System Events" to keystroke "t" using command down'
+
+
+# Under System Preferences select:
+#     > Security & Privacy
+#     > Privacy tab 
+#     > select [Accessibily] in the left panel 
+#     > click the 🔒 icon to unlock and add Bitbar.app, Terminal.app and /usr/bin/osascript
+#     (if the apps are already enabled in the list, then remove them and re-add)
+# In System Settings: allow Terminal full access to Documents (if profiles are in Documents) or full disk access
+
+# START CONFIG
+export PATH="$PATH:/usr/local/bin"
```

Comment:
M1 based Macs install Homebrew to a new location. Not sure if the line below needs to account for this.

```suggestion
export PATH="$PATH:/opt/homebrew/bin:/usr/local/bin"
```